dockerfiles/init/modules/che/templates/che.env.erb

```diff
@@ -58,9 +58,9 @@ JAVA_HTTPS_PROXY_PORT=-Dhttps.proxyPort=<%= @che_https_proxy.gsub(/^https?\:\/\/
 <% end -%>
 <% end -%>
 <% if ! @che_no_proxy.empty? -%>
-JAVA_NO_PROXY=-Dhttp.nonProxyHosts='<%= @che_no_proxy.gsub(/^https?\:\/\//, '').gsub(/^www./,'').gsub(',','|') %>|'
```

Comment:
Looks like this statement can be simplified: 
`.gsub(/^www./,'').split(",").uniq.join("|") %>|'`